bootcamp-milestone-2/src/app/resume/page.tsx

```diff
@@ -0,0 +1,70 @@
+export default function Resume() {
+    return (
+      <main> 
+        <h1 className="page-title"> Resume </h1>
+            <a className="resume-download" href="Elaina Lyons College Resume.pdf"> Download Resume</a>
+            <div className="resume">
+                <section className="section"> 
+                    <h2 className="section-title">Education</h2>
+                    <div className="entry">
+                        <h3 className="entry-title"> <strong> Bachelor of Science in Computer Science</strong></h3>
+                        <p className="entry-info"> California Polytechnic State University, San Luis Obispo | Expected Graduation May 2027</p>
+                    </div>
+                </section>
```

Comment:
You might consider optimizing your code by having a resume section component and pass in the unique data (title, description, etc) as props for each section (experience, education, etc) rather than have a lot of similar JSX code doing in this file.